data_steward/tools/combine_ehr_rdr.py

```diff
@@ -361,10 +381,58 @@ def load_mapped_person():
     query(q, 'person', write_disposition='WRITE_TRUNCATE')
 
 
+def copy_deid_map_table(pid_rid_mapping_table, project_id, dataset_id, client):
+    job_config = bq.bigquery.QueryJobConfig(destination=pid_rid_mapping_table)
+    q = COPY_PID_RID_QUERY.render(
+        project=project_id,
+        dataset=dataset_id,
+        pid_rid_table=combine_consts.PID_RID_MAPPING_TABLE)
+
+    query_job = client.query(q, job_config=job_config)
+    query_job.result()
```

Comment:
This call does block to make sure the query finishes, but you haven't checked to make sure there are no exceptions.
```suggestion
    query_job.result()
    if query_job.exception():
        logging.error(f"The deid_map table did not generate successfully")
```